### PR TITLE
Feature: Slider and Responsive nav

### PR DESCRIPTION
**Pull request by Pratham82**
# Completed Slider and Responsive nav

## Changes made

- [x] Added responsive navbar in all components
- [x] Created slider component
- [x] Added utility classes component

## How does the solution address the problem

This PR will complete the slider mentioned in P2 component

### CODE REVIEW

**Comment by ashifkhn** [comment]
Looks good. Good to go.

File: src/components/navigation/navigation.html [hunk 1]
```diff
@@ -18,6 +24,49 @@
       </div>
     </header>
     <main class="main-container">
+      <div class="respSidebar">
+        <span class="closeNav pr-10 pl-10 text-center" onclick="closeNav()">
+          Close <i class="far fa-times"></i>
+        </span>
+        <h3>Getting started</h3>
+        <nav class="intro-list">
```
Comment (vikrantkode) [inline src/components/navigation/navigation.html:32]: -try using same class name i.e., either camel-casing or - one.
`respSidebar` and `intro-list`

**Review by vikrantkode — COMMENTED** [review]
I checked. all looks good only some places i saw cass name difference pattern. Have a look.
Rest Ui is good

File: src/components/navigation/navigation.html (hunk 1 above)
Comment (Pratham82) [reply src/components/navigation/navigation.html:32]: Thanks, I'll try to implement this

**Pull request merged**